### ISSUE TITLE
Customize loss function with caffe2.python

### ISSUE THREAD

**Issue by zieitokk** [pytorch/pytorch#20479]
Hi I just wondering if there is a way for us to customize loss function by using caffe2 python interface?

If the answer is yes, how could I implement the customized loss function? 

Thank you very much!
